Backend/DataAccessLayer/Repositories/PointRepository.cs

```diff
@@ -0,0 +1,71 @@
+﻿using DataAccessLayer.Database;
+using DataAccessLayer.Models;
+using System;
+using System.Data.Entity;
+using System.Linq;
+
+namespace DataAccessLayer.Repositories
+{
+    public class PointRepository
+    {
+        private Point CheckPointLimits(Point point)
+        {
+            if (point.Longitude > 180 || point.Longitude < -180 ||
+                point.Latitude > 90 || point.Latitude < -90)
+                return null;
+            return point;
+        }
+
+        public bool ExistingPoint(DatabaseContext _db, Point point)
+        {
+            var result = GetPoint(_db, point.Id);
+            if (result != null)
+            {
+                return true;
+            }
+            return false;
+        }
+        public Point CreatePoint(DatabaseContext _db, Point point)
+        {
+            if (CheckPointLimits(point) == null)
+                return null;

```

Comment:
Is this really how we're handling errors?

I'd prefer if we threw an error to be caught by an upper layer. With this, if there are multiple different types of errors, it's impossible for layers above to know what the actual error was for accurate response/reporting.

Reply:
agreed, but we don't have the error handling in place yet, so it is more of a place holder at this point.

Reply:
I've added TODO comments on these lines so we can add the errors later.

Reply:
Can you try to implement some error handling?